configs/common/packages.yaml

```
@@ -107,7 +107,9 @@ packages:
     fms:
       require:
       - '@2024.02'
-      - precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS build_type=Release +deprecated_io
+      - any_of:
+        - precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS build_type=Release +deprecated_io
+        - precision=32,64 +quad_precision ~gfs_phys +openmp +pic constants=GEOS build_type=Release +deprecated_io
```

```suggestion
      - precision=32,64 +quad_precision +openmp +pic build_type=Release +deprecated_io
      - any_of:
        - +gfs_phys constants=GFS
        - ~gfs_phys constants=GEOS
```
I think this should work, please try

---

Ahh. Smart. I'll try this.

---

This worked for the concretizer.